packages/fast-usdc/src/exos/status-manager.js

```diff
@@ -69,6 +69,7 @@ export const prepareStatusManager = (
   txnsNode,
   {
     marshaller,
+    // eslint-disable-next-line no-unused-vars
```

Comment:
we should use the log/tracer, shouldn't we?

Reply:
There's nothing currently logging anything in `statusManager`. vstorage seems to be the main way of communicating